harbors/management/commands/add_helsinki_harbors_images.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from django.conf import settings
+from django.core.files import File
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+
+from harbors.models import Harbor
+
+
+class Command(BaseCommand):
+
+    def handle(self, **options):
+        updated_harbors = 0
+        missing_images = []
+
+        for harbor in Harbor.objects.all():
+            with transaction.atomic():
+                self.stdout.write(
+                    'Assigning image for harbor with servicemap ID {}'.format(harbor.servicemap_id)
+                )
+                image_filename = '{}.jpg'.format(harbor.servicemap_id)
```

Comment:
Should probably first check if `harbor.servicemap_id` is actually set? Or maybe better would be to loop only harbors that have `servicemap_id` set?

Reply:
also good point! will fix this up